openstack/nova/templates/bigvm-deployment.yaml

```diff
@@ -36,38 +36,32 @@ spec:
         {{- end }}
     spec:
 {{ tuple . "nova" "bigvm" | include "kubernetes_pod_anti_affinity" | indent 6 }}
-{{ include "utils.proxysql.pod_settings" . | indent 6 }}
+      {{- include "utils.proxysql.pod_settings" . | indent 6 }}
       terminationGracePeriodSeconds: {{ .Values.defaults.default.graceful_shutdown_timeout | add 5 }}
       hostname: nova-bigvm
+      initContainers:
+      {{- $dependencies := dict }}
+      {{- $_ := set $dependencies "services" (print .Release.Name "-rabbitmq," (include "nova.helpers.database_services" .)) }}
```

Comment:
This is missing the cell2 rabbitmq now :/

Reply:
Changed to all rabbitmqs and dbs (if cell2 is enabled).